pjsip/src/pjsip/sip_dialog.c

```diff
@@ -468,9 +468,10 @@ pj_status_t create_uas_dialog( pjsip_user_agent *ua,
     pj_strdup(dlg->pool, &dlg->remote.info_str, &tmp);
     
     /* Save initial destination host from transport's info */
-    pj_strdup(dlg->pool, &dlg->initial_dest,
-              &rdata->tp_info.transport->remote_name.host);
-
+    if (rdata->tp_info.transport->dir == PJSIP_TP_DIR_OUTGOING) {
+	pj_strdup(dlg->pool, &dlg->initial_dest,
```

Comment:
minor: your editor seems to still use tab?

Reply:
Nice catch! Just updated my editor settings & fixed the patch.